.travis.yml

```diff
@@ -15,17 +15,9 @@ python:
     - 3.4
     - 3.5
 install:
-    - if [[ ${TRAVIS_PYTHON_VERSION%%.*} == '2' ]]; then pip install -r requirements-py2.txt; fi
-    - if [[ ${TRAVIS_PYTHON_VERSION%%.*} == '3' ]]; then pip install -r requirements-py3.txt; fi
-    - python setup.py install
```

Comment:
no need to explicitly install package or dependencies here anymore - tox manages this as part of its configuration / runtime. only required dependencies here are immediate bootstrap packages for running tox and installing dependencies later (setuptools)